addons/mrp/views/mrp_workorder_views.xml

```diff
@@ -105,7 +105,6 @@
         <field name="inherit_id" ref="mrp.mrp_production_workorder_tree_editable_view"/>
         <field name="arch" type="xml">
             <xpath expr="//tree" position="attributes">
-                <attribute name="create">0</attribute>
```

Comment:
Seems a bit risky for a stable. Since we are not sure every flow would work correctly. And we expect people to fill their workorders inside the production form view. I would do it only on the customer instance